src/styles/components/button.scss

```diff
@@ -1,6 +1,4 @@
 .v-btn {
-  width: 340px;
-
```

Comment:
Ist die Änderung hier in dem File noch notwendig, jetzt, dass wir kein Datepicker nutzen?

Reply:
war anscheinend noch nicht gepushed... ist raus!